x-pack/plugins/infra/public/test_utils/source_configuration.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const DEFAULT_SOURCE_CONFIGURATION = {
```

Comment:
Do we want to type this so we don't forget to update it when the source shape changes?